### PR TITLE
[delete Product] creando ruta delete

### PR DESCRIPTION
**Pull request by nestornovella**
DESCRIPTION 

crando ruta delete ....

### CODE REVIEW

File: api/controllers/Product/deleteProduct.js [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+const { endpointResponse } = require('../../helpers/success');
+const { ErrorObject } = require('../../helpers/error');
+
+module.exports = {
+    deleteProduct: async (req, res, next) => {
+        const { productId } = req.params;
+        try {
+            if(!productId)throw new ErrorObject("missing params", 404)
+            const deleteProduct = await Product.destroy({ where: { id: productId } })
+            if(deleteProduct === null) throw new ErrorObject("missing params", 404)
```
Comment (miguelbel00) [inline api/controllers/Product/deleteProduct.js:12]: error message change for "product not found"

**Review by miguelbel00 — CHANGES_REQUESTED** [review]

**Comment by miguelbel00** [comment]
Is lacking postman evidence

**Review by miguelbel00 — APPROVED** [review]
All ok

**Review by agustinhummel — APPROVED** [review]
okok

**Pull request merged**